services/venice-controller/src/test/java/com/linkedin/venice/controller/server/TestAdminSparkServer.java

```diff
@@ -1000,7 +1000,8 @@ public void testCleanupInstanceCustomizedStates() {
     childControllerAdmin.incrementVersionIdempotent(clusterName, storeName, "test", 1, 1);
     MultiStoreTopicsResponse response = controllerClient.cleanupInstanceCustomizedStates();
     Assert.assertFalse(response.isError());
-    Assert.assertTrue(response.getTopics().isEmpty());
+    Assert.assertNotNull(response.getTopics());
+    Assert.assertFalse(response.getTopics().contains(storeName));
```

Comment:
This will always be `false`, right since store name and topic names are different

Reply:
Oh yeah, my bad.  Good catch.  Thanks.